### ISSUE TITLE
Error, mis-combining components

### ISSUE THREAD

**Issue by drmcnelson** [SchrodingersGat/KiBoM#187]
The following two items from the schematic are 30k resistots, but the first is  0.1%, 1/5W,  the second is 1% 1/4W.

![image](https://github.com/SchrodingersGat/KiBoM/assets/38619857/79296304-954e-4847-976e-d2de32e61d6e)

![image](https://github.com/SchrodingersGat/KiBoM/assets/38619857/570a1aba-d744-4215-b3b6-f624720581d0)

KiBoM  combines them into one spreadsheet row,  and puts both partnumbers in the digikey column. 

![image](https://github.com/SchrodingersGat/KiBoM/assets/38619857/6a261512-da69-4a7f-8e18-19ae6ca9b503)

Please advise.

**Comment by drmcnelson** [comment]
Ok,  so I found the bom.ini and added a field "Description".

Now, the issue is a default location for the bom.ini,   perhaps .bom.ini in the login directory

**Comment by SchrodingersGat** [comment]
Yep that a [.ini configuration option](https://github.com/SchrodingersGat/KiBoM#custom-column-grouping)

> Now, the issue is a default location for the bom.ini, perhaps .bom.ini in the login directory

? A default .ini file should be created already?

**Comment by drmcnelson** [comment]
I found the default .ini in the project directory.    That is not so convenient.   I think what you want is the following (in schematic)

 if --cfg specified
   use it,
else if in project directory
  use it,
else if .kibom.ini in login directory or under .config/kibom/kibom.ini   
  use it
else 
   create it in local project directory and use it

**Comment by drmcnelson** [comment]
BTW,  the program works well.   Very helpful.
